includes/class-wc-payments-account.php

```diff
@@ -643,4 +648,29 @@ private function settings_changed( $changes = [] ) {
 		$diff = array_diff_assoc( $changes, $account );
 		return ! empty( $diff );
 	}
+
+	/**
+	 * Check if the named note has been actioned.
+	 *
+	 * @param string $note_name The name of the note to check.
+	 *
+	 * @return bool Flag indicating whether the note has been actioned.
+	 */
+	private function note_actioned( string $note_name ) {
+		// TODO: Can we look-up by name?
+		// TODO: WC compatibility, I think some class names changed?
```

Comment:
I _think_ we could/should keep using this name – will be deprecated soon but isn't yet (pbIJXs-qi-p2).

Reply:
Thanks for the link. I've confirmed that these classes are available on our earliest supported WC version (4.0). I'll create an issue so that we can write some version specific code for WC 4.8 later.